Src/Support/Google.Apis.Auth/OAuth2/Flows/AuthorizationCodeFlow.cs

```diff
@@ -182,18 +208,34 @@ public AuthorizationCodeFlow(Initializer initializer)
             scopes = initializer.Scopes;
 
             // Set the HTTP client.
-            var httpArgs = new CreateHttpClientArgs();
+            DefaultExponentialBackOffPolicy = initializer.DefaultExponentialBackOffPolicy;
+            HttpClientFactory = (initializer.HttpClientFactory ?? new HttpClientFactory());
 
-            // Add exponential back-off initializer if necessary.
-            if (initializer.DefaultExponentialBackOffPolicy != ExponentialBackOffPolicy.None)
+            if (initializer.HttpClient != null)
+            {
+                httpClient = initializer.HttpClient;
```

Comment:
Is the HttpClientFactory used at all in this case? If not, it might be worth moving the assignment to the else block. I think I've missed some of the context though - hopefully it'll be more obvious as I go.